referral/templates/referral/new-patient-contact.html

```diff
@@ -1,24 +1,27 @@
 {% extends "pttrack/base.html" %}
 
+{% load i18n %}
+<!-- trans finished -->
+
 {% block title %}
-New referral followup
+{% blocktrans %}New referral followup{% endblocktrans %}
 {% endblock %}
 
 {% block header %}
-<h1>New referral followup</h1>
+<h1>{% blocktrans %}New referral followup{% endblocktrans %}</h1>
 {% if patient %}
-<p class="lead">Referral for <em><a href="{% url 'patient-detail' pk=patient.id %}">{{ patient.name }}</a></em></p>
+<p class="lead">{% blocktrans %}Referral for{% endblocktrans %} <em><a href="{% url 'patient-detail' pk=patient.id %}">{{ patient.name }}</a></em></p>
 {% endif %}
-<p class="lead">Referral to 
+<p class="lead">{% blocktrans %}Referral to{% endblocktrans %}
```

Comment:
I think the `blocktrans` directive [supports putting django template language stuff _inside_ it](https://docs.djangoproject.com/en/1.11/topics/i18n/translation/#blocktrans-template-tag), so you could have this be all one big blocktrans, no?